### DISCUSSION TITLE
Revise default anti-cheat thresholds

### DISCUSSION THREAD

**Discussion by moisterrific** [Pryaxis/TShock#1934] [Ideas]
The current values are too low and will definitely be producing false positives in a normal playthrough. But setting them too high will make it easier for cheaters to exploit. 

What do you think should be appropriate values for each of the thresholds? 

Current defaults:
HealOther = 50
Projectile = 50
TileKill = 60
TileLiquid = 15
TilePaint = 15
TilePlace = 20

**Comment by hakusaro** [comment]
I'd love it if you can come up with good values for this. Once upon a time we did a survey to ask people what the values should be, but we didn't get much traction.

**Reply by moisterrific** [reply]
I’ll do some playthroughs with 2x or higher values and see.

**Reply by HeySteets** [reply]
+1 for doubling the defaults - Even at triple and quadruple, players still sometimes hit false positives with the newer, higher-powered weapons.

**Reply by hakusaro** [reply]
I'd love to see the results of playing with 2x or higher.

**Reply by moisterrific** [reply]
I’ll try with 2x as a baseline and slightly raise it if/when we run into some thresholds.

**Comment by k8ieone** [comment]
I assume the values are the same for vanilla Terraria server. We played with my friend a couple of days back and during a pirate invasion he was using bee grenades. He got booted quite a few times. To be clear, I have `secure=1` in my server config and I think If I would remove that, it would work just fine.

**Reply by hakusaro** [reply]
Vanilla does not have these checks...

**Reply by k8ieone** [reply]
It does! Removing `secure=1` helped. Probably something new. Anyway... sorry for bringing it here.
